src/index.html

```diff
@@ -5,8 +5,425 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+          >
```

Comment:
```suggestion
        <img
          src="images/logo.png"
          alt="logo"
          class="header__logo"
        >
```
Closing tag should be under the opening